channels/se.m3u

```diff
@@ -49,3 +49,5 @@ http://195.189.105.158:9050/134/index.m3u8
 https://edg01-prd-se-dcs.solidtango.com/edge/_lo9yf4l5_/lo9yf4l5/playlist.m3u8
 #EXTINF:-1 tvg-id="OppnaKanalen.se" tvg-name="Öppna Kanalen" tvg-country="SE" tvg-language="Swedish" tvg-logo="https://i.imgur.com/O1wCZTZ.jpg" group-title="Local",Öppna Kanalen (540p)
 http://83.140.64.214/edge/_451iw2h_/451iw2h/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV10
```

Comment:
Not quite sure that's the correct country. I see CBN News
![vlcsnap-2021-06-07-21h19m10s277](https://user-images.githubusercontent.com/30985701/121075884-03772200-c7d6-11eb-93cd-13aad07fa079.png)

Reply:
![image](https://user-images.githubusercontent.com/5430116/121078743-01b35b80-c7e2-11eb-98c3-30d45c0528c5.png)
It was reffering CBN News

Reply:
@LaneSh4d0w

Reply:
https://en.wikipedia.org/wiki/TV10_(Sweden)

TV10 doesn't have this logo, it's this one : https://upload.wikimedia.org/wikipedia/en/7/7a/TV10_logo_2010_better.JPG

Reply:
Found one, religious channel @LaneSh4d0w  https://www.lyngsat.com/tvchannels/se/Kanal-10-Sverige.html
https://sv.wikipedia.org/wiki/Kanal_10